packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -521,6 +553,106 @@ describe('Checkout', () => {
                 expect(result).toBe(false);
             });
         });
+
+        describe('shouldShowErrorDialog ::', () => {
+            it('should return `true` if `nonFulfillableError.showInDialog` is `true`', () => {
+                // Arrange
+                const wrapper = shallowMount(VueCheckout, {
+                    store: createStore({
+                        ...defaultCheckoutState,
+                        isLoggedIn: true
+                    }),
+                    i18n,
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            nonFulfillableError: {
+                                code: ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE,
+                                showInDialog: true
+                            }
+                        };
+                    }
+                });
+
+                // Act
+                const result = wrapper.vm.shouldShowErrorDialog;
+
+                // Assert
+                expect(result).toBe(true);
+            });
+
+            it('should return `false` if `nonFulfillableError.showInDialog` is `false`', () => {
+                // Arrange
+                const wrapper = shallowMount(VueCheckout, {
+                    store: createStore({
+                        ...defaultCheckoutState,
+                        isLoggedIn: true
+                    }),
+                    i18n,
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            nonFulfillableError: {
+                                code: ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE,
+                                showInDialog: false
+                            }
+                        };
+                    }
+                });
+
+                // Act
+                const result = wrapper.vm.shouldShowErrorDialog;
+
+                // Assert
+                expect(result).toBe(false);
+            });
+
+            it('should return `null` if there is no error', () => {
```

Comment:
Is there any reason to return `null` in this case? If there is no error, `shouldShowErrorDialog` should be `false`, right?